docs/references/configuration.rst

```diff
@@ -146,6 +146,21 @@ To reload the configuration from within the database, you can use the ``NOTIFY``
 List of parameters
 ==================
 
+.. _admin-server-host:
+
+admin-server-host
+-----------------
+
+  =============== =======================
+  **Type**        String
+  **Default**     `server-host` value
+  **Reloadable**  N
+  **Environment** PGRST_ADMIN_SERVER_HOST
+  **In-Database** `n/a`
+  =============== =======================
+
+  Specifies the host for the :ref:`admin_server`. Defaults to :ref:`server-host` value.
```

Comment:
Here, I understand that if I set only the `server-host`, like:

```
server-host = "127.0.0.1"
```

Then the `admin-server-host` should also be `127.0.0.1`?  In the code I see that it defaults to the fallback `!4` not the `server-host` right now.

If so, it would also need a test.

Reply:
@wolfgangwalther good call, implemented it.